sdk/couchbase-core/src/mgmtoptions.rs

```diff
@@ -579,3 +580,426 @@ impl<'a> EnsureBucketOptions<'a> {
         self
     }
 }
+
+#[derive(Debug, Clone)]
+#[non_exhaustive]
+pub struct GetUserOptions<'a> {
+    pub username: &'a str,
+    pub domain_name: &'a str,
```

Comment:
`domain_name` seems like the wrong name for this. Shouldn't it just be `domain` or `auth_domain` (for consistency with rfc)? Otherwise it sounds like the internet domain kind of domain